dldashboard/web/src/StateManager.js

```diff
@@ -53,6 +54,7 @@ class StateManager {
       { msg: "", failed: false },
       { msg: "", failed: false },
     ],
+    handshake: "",
```

Comment:
StateManager maintains all states for various components. If you need to put a field here, make sure to prefix it with component name, i.e. timeline.